deploy/99_multisig.ts

```diff
@@ -25,11 +26,21 @@ const func: DeployFunction = async function (hre1: HardhatRuntimeEnvironment) {
 
   const network: Network = networks[hre.networkName];
 
-  if (network.type == NetworkType.mainnet) {
+  const environment: Environment = getEnvironment();
+
+  if (
+    (network.type == NetworkType.mainnet || network.type == NetworkType.testnet) &&
```

Comment:
@alexanderattar @ACC01ADE given that we cant do mutli sig for all networks right now. should we remove the testnet check?